Branch-SDK/Branch.h

```diff
@@ -676,6 +676,13 @@ typedef NS_ENUM(NSUInteger, BranchCreditHistoryOrder) {
  */
 - (void)ignoreAppleSearchAdsTestData;
 
+/**
+ Set the AppGroup used to share data between the App Clip and the Full App.
+ 
+ This must be set before initSession is called.
+ */
+- (void)setAppClipAppGroup:(NSString *)appGroup;
```

Comment:
This seems like the kind of thing we could easily support via configuration as well, e.g. Info.plist (or branch.json).

Reply:
Yea agree.